rust/src/nfs/nfs2_records.rs

```diff
@@ -96,3 +96,211 @@ pub fn parse_nfs2_attribs(i: &[u8]) -> IResult<&[u8], Nfs2Attributes> {
     let attrs = Nfs2Attributes { atype, asize };
     Ok((i, attrs))
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::nfs::nfs2_records::*;
+    use nom7::bytes::streaming::take;
+    use nom7::IResult;
+
+    #[test]
+    fn test_nfs2_handle() {
+
```

Comment:
not sure if a separate `test_nfs2_handle()` is needed, since we already use `parse_nfs2_handle()` in later tests?

Reply:
This seems good to me since it is a separate unit too.